tests/_tools/tools.js

```diff
@@ -67,4 +67,29 @@
 			return count;
 		}
 	};
+
+	bender.tools.operations = {
+		expectOperation: ( Position, Range ) => {
```

Comment:
I agree, previously it was used in multiple files, I forgot to bring it back from `tools.js`.